rustls/Cargo.toml

```diff
@@ -61,6 +61,8 @@ zlib = ["dep:zlib-rs"]
 
 # FEATURES to help build for no-std targets with no atomic ptr - see documentation for more info
 critical-section = ["dep:portable-atomic", "once_cell/portable-atomic", "portable-atomic/critical-section"]
+# TODO: ADD EXPLICIT "dep:portable-atomic" ENTRY for unsafe-assume-single-core as well for extra clarity ref:
+# - https://github.com/brody4hire/portable-rustls/issues/54
 unsafe-assume-single-core = ["once_cell/portable-atomic", "portable-atomic/unsafe-assume-single-core"]
```

Comment:
I think "once_cell/portable-atomic" can be removed from both of these options, as `once_cell` with `portable-atomic` feature is now enabled regardless of features selected.

Alternative may be to add a TODO comment for `once_cell` below that use with `portable-atomic` should be optional, with feature gate.